Makefile

```diff
@@ -0,0 +1,198 @@
+# Update this value when you upgrade the version of your project.
+VERSION ?= v0.0.1
```

Comment:
should this be 0.1.0?

Reply:
I'm thinking v0.0.1 for alpha, v0.1.0 for beta and v1.0.0 for GA. wdyt?